packages/teams-js/src/private/externalAppCardActions.ts

```diff
@@ -0,0 +1,169 @@
+import { sendMessageToParentAsync } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { FrameContexts } from '../public';
+import { errorNotSupportedOnPlatform } from '../public/constants';
+import { runtime } from '../public/runtime';
+
+/**
+ * @hidden
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export namespace externalAppCardActions {
+  /**
+   * @hidden
+   * @internal
+   * Limited to Microsoft-internal use
+   */
+  export enum ActionOpenUrlType {
+    DeepLinkOther,
```

Comment:
I think it would be safer to define these as strings here and in the host, since I think by default these compile to integers, right?

Reply:
I think by default they are integers/numbers.